src/xsar/rcm_meta.py

```diff
@@ -103,7 +106,7 @@ def flip_sample_da(self):
         antenna_pointing = self.dt['sourceAttributes/radarParameters'].attrs['antennaPointing']
         pass_direction = self.dt['sourceAttributes/orbitAndAttitude/orbitInformation'].attrs['passDirection']
         flipped_cases = [('Left', 'Ascending'), ('Right', 'Descending')]
-        samples_depending_ds = ['geolocationGrid', 'lut', 'noise_lut']
+        samples_depending_ds = ['geolocationGrid']
```

Comment:
Is it the modification tha prevent the bug of flipped RCM images? Is there an issue opened related to this problem?

Reply:
No issue, should I create one to explain the problem? 
Yes, this is the line that prevent the RCM bug

Reply:
note: the variable name `samples_depending_ds` is not super explicit.